src/v2/base.ts

```diff
@@ -54,6 +54,31 @@ export class BaseAPI {
             this.configuration = configuration;
             this.basePath = configuration.basePath || this.basePath;
         }
+
+        axios.interceptors.response.use(
```

Comment:
Can we wrap this into a function and call the function instead of c/p to every api version?

Reply:
good point!